coruscant/src/components/LoginPrompt.js

```diff
@@ -0,0 +1,50 @@
+import React, {Component} from 'react';
+import RaisedButton from 'material-ui/RaisedButton';
+import TextField from 'material-ui/TextField';
+
+class Login extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            username: '',
+            password: ''
+        }
+    }
+
+    handleClick(event) {
+        var self = this;
+        console.log({
+            "username": this.state.username,
+            "password": this.state.password
```

Comment:
Remove the password being logged to the console. Just so there's no chance it gets left in later.